src/routing/bootstrap.rs

```diff
@@ -21,18 +21,19 @@ use crate::{
 };
 use bytes::Bytes;
 use futures::future;
+use resource_proof::ResourceProof;
 use std::{collections::VecDeque, mem, net::SocketAddr};
 use tokio::sync::mpsc;
 use xor_name::Prefix;
 
 const BACKLOG_CAPACITY: usize = 100;
 
-/// Bootstrap into the network as an infant node.
+/// Bootstrap into the network as an adult node.
 ///
 /// NOTE: It's not guaranteed this function ever returns. This can happen due to messages being
 /// lost in transit or other reasons. It's the responsibility of the caller to handle this case,
 /// for example by using a timeout.
-pub(crate) async fn infant(
+pub(crate) async fn adult(
```

Comment:
The name of this function is confusing now, because adult can be both joining for the first time or being relocated. This name should reflect that this is a bootstrap as a new node as opposed to bootstrap as relocated node. Some suggestions: `first`, `new` (or `new_node`), `initial`, ... something like that.